src/styles/use_case.css

```diff
@@ -0,0 +1,62 @@
+.use-case-page {
+  .title {
+    font-size: 32px;
+    font-weight: bold;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  .col-12 {
+    padding: 20px !important;
```

Comment:
why `!important`, which style is it overriding?

Reply:
@pkong-ds to override bootstrap `col-12` default padding